docs/api.md

```diff
@@ -216,13 +216,10 @@ curl -v \
 
 ### Response
 
-Successful requests will produce a "200 OK" response with the lockout
-advice in the JSON body:
+Successful requests will produce a "200 OK" response:
 
 ```json
-{
-  "lockout": false
-}
+{}
```

Comment:
not sure this is what we want but it looks like this is what the code is doing on: https://github.com/mozilla/fxa-customs-server/blob/master/lib/server.js#L305